test/require-test.js

```diff
@@ -0,0 +1,47 @@
+var path = require('path');
+var temp = require('temp');
+var mocha = require('../');
+var fs = require('fs');
+var assert = require('assert');
+
+var tempFile, filePrefix = './';
+
+function removeFile(file) {
+  if (file && fs.existsSync(file)) { 
+    fs.unlinkSync(file); 
+  };
+}
+
+beforeEach(function(){
+  tempFile = temp.path({dir:process.cwd(), suffix: '.js'});
+  tempFileBaseName = path.basename(tempFile);
+  fs.closeSync(fs.openSync(tempFile, 'w'));
```

Comment:
I just want an empty file in the current working directory. temp-write seems to write the the tmp-directory of the running OS.

Importing the file using the whole path works before https://github.com/sindresorhus/gulp-mocha/commit/044d6b67a3beced6da601c695e240e78ab96f5ad so I don't feel that the test would cover the case I was having trouble with.